HopsanCore/include/ComponentUtilities/WhiteGaussianNoise.h

```diff
@@ -42,7 +42,7 @@ namespace hopsan {
     class HOPSANCORE_DLLAPI WhiteGaussianNoise
     {
     public:
-        double getValue();
+        static double getValue();
```

Comment:
Good, but this should not even be a class, just a function would do. But that could break stuff so lets keep it this way for now

Reply:
I agree. The problem is if someone is using this in their own libraries. We could keep it as a class and add standard deviation as a member, since you normally want to rescale the noise anyway. But that does not belong in this PR.